xss_test.go

```diff
@@ -29,6 +30,11 @@ func TestIsXSS(t *testing.T) {
 		"<a href  =   javascript:alert(1); >",
 		"<a href=\"  javascript:alert(1);\" >",
 		"<a href=\"JAVASCRIPT:alert(1);\" >",
+		"<style>@keyframes x{}</style><xss style=\"animation-name:x\" onanimationstart=\"alert(1)\"></xss>",
```

Comment:
Thanks also for extending the tests! Considering that we are closing a false positive report would be good to have also some tests that we expect to not match, but it would require refactoring a bit this tests setting of we are expecting a match for each payload

Reply:
Refactored and added negative tests.